### DISCUSSION TITLE
init_from_config

### DISCUSSION THREAD

**Discussion by p0p4k** [coqui-ai/TTS#1488] [General Q&A] [answered]
https://github.com/coqui-ai/TTS/blob/0cf3265a4686d7e856bd472cdaf1572d61cab2b8/TTS/model.py#L16

https://github.com/coqui-ai/TTS/blob/0cf3265a4686d7e856bd472cdaf1572d61cab2b8/TTS/model.py#L141

Noob python implementation question here. Why do we have 2 methods with same name in the BaseTrainerModel? Isnt the second one sufficient? Thanks.

**Comment by erogol** [comment, marked as answer]
Yep, it is redundant. We'll remove the second one :)

**Reply by p0p4k** [reply]
Ok. Thanks once again for the good work! Your blogs have been great source of inspiration for me!!
